src/Dashboard.css

```diff
@@ -0,0 +1,13 @@
+.dashboard {
+  height:100%;
+  width:100%;
+  bottom:0;
+}
+
+.dashboard-div {
```

Comment:
would prefer calling this `-container` or `-wrapper` as opposed to `-div` which is really opinionated in what type of element this should be